instances/actionStatusType/completed.jsonld

```diff
@@ -4,11 +4,14 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/actionStatusType/completed",
   "@type": "https://openminds.ebrains.eu/controlledTerms/ActionStatusType",
-  "definition": "An action that has already taken place",
+  "definition": "An action that has already taken place.",
```

Comment:
I think some reference to the success of the action is needed, to distinguish it from "failed", which is also an action that has already taken place.